scripts/install_moodle.sh

```diff
@@ -243,177 +269,189 @@ echo_info "Done."
 #           available through lsblk, thus we wait and loop for up to 60 seconds to get it.
-elapsedTime=0
-dataDiskFileSystemUuid=""
-while [ -z "$dataDiskFileSystemUuid" -a "$elapsedTime" -lt "60" ]; do
-    echo_info "Waiting for 1 second..."
-    sleep 1
-    dataDiskFileSystemUuid=$(lsblk --noheadings --output UUID ${dataDiskBlockPath})
-    ((elapsedTime+=1))
+elapsed_time=0
+data_disk_file_system_uuid=""
+while [[ -z "${data_disk_file_system_uuid}" && "${elapsed_time}" -lt "60" ]]; do
+  echo_info "Waiting for 1 second..."
+  sleep 1
+  data_disk_file_system_uuid=$(lsblk --noheadings --output UUID ${data_disk_block_path})
+  ((elapsed_time+=1))
 done
-echo_info "Data disk file system UUID: $dataDiskFileSystemUuid"
+echo_info "Data disk file system UUID: ${data_disk_file_system_uuid}"
 echo_info "Done."
 
 echo_action "Creating Moodle Data mount point..."
 mkdir -p ${parameters[moodleDataMountPointPath]}
 echo_info "${parameters[moodleDataMountPointPath]} directory created."
 echo_info "Done."
 
-fstabFilePath=/etc/fstab
-echo_action "Updating $fstabFilePath file to automount the data disk using its UUID..."
-if ! grep -q "$dataDiskFileSystemUuid" $fstabFilePath; then
-    printf "UUID=${dataDiskFileSystemUuid}\t${parameters[moodleDataMountPointPath]}\t${dataDiskFileSystemType}\tdefaults,nofail\t0\t2\n" >> $fstabFilePath
-    echo_info "Done."
+fstab_file_path=/etc/fstab
+echo_action "Updating $fstab_file_path file to automount the data disk using its UUID..."
+if ! grep -q "${data_disk_file_system_uuid}" ${fstab_file_path}; then
+  printf "UUID=${data_disk_file_system_uuid}\t${parameters[moodleDataMountPointPath]}\t${data_disk_file_system_type}\tdefaults,nofail\t0\t2\n" >> ${fstab_file_path}
+  echo_info "Done."
 else
-    echo_info "Skipped: already set up."
+  echo_info "Skipped: already set up."
 fi
 
 echo_action 'Mounting all drives...'
 mount -a
 echo_info "Done."
 
 echo_action 'Setting permissions ...'
-chown -R ${apache2User}:root ${parameters[moodleDataMountPointPath]}
+chown -R ${APACHE2_USER}:root ${parameters[moodleDataMountPointPath]}
 chmod -R 775 ${parameters[moodleDataMountPointPath]}
 echo_info "Done."
 
 ###############################################################################
 echo_title "Create Moodle Local Cache directory."
 ###############################################################################
-if [ -d ${moodleLocalCacheRootDirPath} ]; then
-    echo_action "Deleting old ${moodleLocalCacheRootDirPath} folder..."
-    rm -rf ${moodleLocalCacheRootDirPath}
-    echo_info "Done."
+if [ -d ${MOODLE_LOCAL_CACHE_DIR_PATH} ]; then
+  echo_action "Deleting old ${MOODLE_LOCAL_CACHE_DIR_PATH} folder..."
+  rm -rf ${MOODLE_LOCAL_CACHE_DIR_PATH}
+  echo_info "Done."
 fi
 
-echo_action "Creating new ${moodleLocalCacheRootDirPath} folder..."
-mkdir ${moodleLocalCacheRootDirPath}
+echo_action "Creating new ${MOODLE_LOCAL_CACHE_DIR_PATH} folder..."
+mkdir ${MOODLE_LOCAL_CACHE_DIR_PATH}
```

Comment:
How about the install command to create the folder and setting up the owner at the same time?
install -o ${APACHE2_USER} -d ${MOODLE_LOCAL_CACHE_DIR_PATH}

Reply:
Interesting.